src/common/hotbar-store.ts

```diff
@@ -35,10 +35,14 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
   }
 
   @action protected async fromStore(data: Partial<HotbarStoreModel> = {}) {
-    this.hotbars = data.hotbars || [{
-      name: "default",
-      items: []
-    }];
+    if (data.hotbars?.length === 0) {
+      this.hotbars = [{
+        name: "default",
```

Comment:
Is the name the same as ID? Are we allowed to delete the default hotbar? Or is that for future considerations?

Reply:
See #2552 (which is follow-up for this PR).